### DISCUSSION TITLE
How to pass migration action, strategy and provider etc., while import and export

### DISCUSSION THREAD

**Discussion by PavanKalyan-Gandhapudi** [keycloak/keycloak#19699] [Q&A]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

import-export

### Describe the bug

I need to pass keycloak migration strategy, provider, usersExportStrategy, realmName while importing. Is there any way how to pass this as a parameters while importing/exporting.

In older keycloak version 19.0 they are passing like below :
1. -Dkeycloak.migration.provider=singleFile
2. -Dkeycloak.migration.action=export
3. -Dkeycloak.migration.usersExportStrategy=REALM_FILE
4. -Dkeycloak.migration.realmName=DataProtector
5. -Dkeycloak.migration.file=C:\dataExport.json
6. -Dkeycloak.migration.strategy=OVERWRITE_EXISTING

### Version

21.0.2

### Expected behavior

Need to pass the above parameters while import/export.

### Actual behavior

As of i observered, didn't find the parameters like what we have in keycloak19.

### How to Reproduce?

Import Realm data by passing above mentioned parameters

**Comment by hmlnarik** [comment]
Converted to a discussion since this is not a bug but a question.
Please see https://www.keycloak.org/server/importExport

**Reply by PavanKalyan-Gandhapudi** [reply]
Hi @hmlnarik , From that link i can see there is no where mentioned how to pass the above parameters like usersExportStrategy, provider etc., Any comments on this ?

**Reply by outsinre** [reply]
I have a similar inquery.

**Reply by belisariops** [reply]
Have the same problem, the provided documentation does not help.
